src/ui/buttons/buttonDesigns/resizeButtonShowLabelText.jsx

```diff
@@ -0,0 +1,196 @@
+/* @flow */
+/** @jsx node */
+import { PPLogo } from '@paypal/sdk-logos/src';
+import { node, Fragment, type ChildType } from 'jsx-pragmatic/src';
+
+import { DESIGN_SMALL_BUTTON_CONFIG } from './constants';
+import type { ContentOptions } from './types';
+
+type AnimationProps = {|
+    designContainer : Object,
+    paypalLabelContainerElement : Object,
+    labelFontSize : number,
+    marginLabelContainer : number,
+    buttonHeight : number,
+    topPositionBlueLayer : string
+|};
+
+export function resizeButtonShowLabelTextComponent({ designLabelText, logoColor } : ContentOptions) : ChildType {
+    // experimentName must match elmo experiment name
+    const ANIMATION_LABEL = DESIGN_SMALL_BUTTON_CONFIG.cssClasses.ANIMATION_LABEL_CONTAINER;
+    const ANIMATION_CONTAINER = DESIGN_SMALL_BUTTON_CONFIG.cssClasses.ANIMATION_CONTAINER;
+    return (
+        <Fragment>
+            <PPLogo logoColor={ logoColor } />
+            <div class='blue-layer' />
+            <div class={ ANIMATION_LABEL } data-design-experiment='104519'>
+                <span>{designLabelText}</span>
+            </div>
+            <style innerHTML={ `
+                    .${ ANIMATION_CONTAINER } .blue-layer {
+                        position: fixed;
+                        opacity: 0;
+                    }
+                    .${ DESIGN_SMALL_BUTTON_CONFIG.cssClasses.DOM_READY } .${ ANIMATION_CONTAINER } img.${ DESIGN_SMALL_BUTTON_CONFIG.cssClasses.PAYPAL_LOGO }-pp{
+                        position: fixed;
+                        opacity:0;
+                    }
+                    .${ ANIMATION_CONTAINER } .${ ANIMATION_LABEL } {
+                        opacity: 0; 
+                        color: white;
+                        font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+                    }
+                ` } />
+        </Fragment>
+    );
+}
+
+// Returns label container if the button sizes match
+export const resizeButtonShowLabelTextProps = function(document : Object, configuration : Object) : AnimationProps | null {
+    let labelFontSize = 8;
```

Comment:
Is this necessary?

Reply:
yeah this is the initial value, and depending on the buttons width it can change ( see line 71 )